<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>504</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>100</red>
        <green>100</green>
        <blue>100</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>100</red>
        <green>100</green>
        <blue>100</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>100</red>
        <green>100</green>
        <blue>100</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>100</red>
        <green>100</green>
        <blue>100</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>100</red>
        <green>100</green>
        <blue>100</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>100</red>
        <green>100</green>
        <blue>100</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>100</red>
        <green>100</green>
        <blue>100</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>100</red>
        <green>100</green>
        <blue>100</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>100</red>
        <green>100</green>
        <blue>100</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(100,100,100)</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>741</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
background:qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 0, 0, 255), stop:0.05 rgba(14, 8, 73, 255), stop:0.36 rgba(28, 17, 145, 255), stop:0.6 rgba(126, 14, 81, 255), stop:0.75 rgba(234, 11, 11, 255), stop:0.79 rgba(244, 70, 5, 255), stop:0.86 rgba(255, 136, 0, 255), stop:0.935 rgba(239, 236, 55, 255));
font: bold italic large &quot;Monaco&quot;;
}
QPushButton{
    border-radius: 10px;
    background: white;
    min-width: 70px;
}
QPushButton:hover{
background:rgba(255, 136, 0, 255);
}
QPushButton:pressed{
background:rgb(255, 150, 255);
}
QLabel{
	background:rgba(255,254,242,0.5);
	border: 2px solid rgba(255,254,242,1);
    border-radius: 15px;
	color: black;
	font-size:13px;
}
QLineEdit{
	background:rgba(255,254,242,0.5);
}
</string>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font-size: 30px;
}</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>471</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	border: none;
	font-size: 35px;
	color: rgb(22, 13, 116);
}</string>
      </property>
      <property name="text">
       <string>  APLIKACJA JĘZYKOWA</string>
      </property>
     </widget>
     <widget class="QPushButton" name="createTestBtn">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>230</y>
        <width>250</width>
        <height>150</height>
       </rect>
      </property>
      <property name="text">
       <string>Stwórz test</string>
      </property>
     </widget>
     <widget class="QPushButton" name="listBtn">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>230</y>
        <width>250</width>
        <height>150</height>
       </rect>
      </property>
      <property name="text">
       <string>Wybierz test</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="WordCompare">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>180</y>
        <width>181</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>Porównywanie słów</string>
      </property>
     </widget>
     <widget class="QPushButton" name="EnglishBtn">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>110</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
    border-radius: 10px;
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 white, stop:0.33 white, stop:0.34 red, stop:0.66 red, stop:0.67 white, stop:1 white); 
    min-width: 70px;
}
QPushButton:hover{
border:  2px solid black; 
}
QPushButton:pressed{
background:black;
color:white;
}</string>
      </property>
      <property name="text">
       <string>ANGIELSKI</string>
      </property>
     </widget>
     <widget class="QPushButton" name="DeutschBtn">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>110</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{background:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:0.33 rgba(0, 0, 0, 255), stop:0.34 rgba(255, 30, 30, 255), stop:0.66 rgba(255, 0, 0, 255), stop:0.67 rgba(255, 255, 0, 255), stop:1 rgba(255, 255, 0, 255));
}
QPushButton:hover{
border:  2px solid black; 
}
QPushButton:pressed{
background:black;
color:white;
}</string>
      </property>
      <property name="text">
       <string>NIEMIECKI</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>100</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Monaco</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string> Lista dostępnych testów</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>400</y>
        <width>151</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Powrót</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_test">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>180</y>
        <width>201</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #ffffff;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="start_test">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>400</y>
        <width>151</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Rozpocznij </string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QPushButton" name="CheckBtn1">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>170</y>
        <width>150</width>
        <height>100</height>
       </rect>
      </property>
      <property name="text">
       <string>Sprawdź</string>
      </property>
     </widget>
     <widget class="QPushButton" name="RandomWord">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>170</y>
        <width>150</width>
        <height>100</height>
       </rect>
      </property>
      <property name="text">
       <string>Losowe słowo</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ShowText">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>90</y>
        <width>200</width>
        <height>50</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="TypeText">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>90</y>
        <width>200</width>
        <height>50</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="MainMenu">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>130</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Menu Główne</string>
      </property>
     </widget>
     <widget class="QLabel" name="pbAnswer">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>320</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QPushButton" name="MainMenu_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>130</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Menu Główne</string>
      </property>
     </widget>
     <widget class="QLabel" name="imageLabel">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>70</y>
        <width>321</width>
        <height>201</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="ImgBtn">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>290</y>
        <width>97</width>
        <height>33</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Losuj</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="picText">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>150</y>
        <width>130</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="CheckBtn2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>190</y>
        <width>130</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Sprawdź</string>
      </property>
     </widget>
     <widget class="QLabel" name="ImgAnswer">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>350</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QPushButton" name="jpnRand">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Losuj</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="jpnTranslate">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>210</y>
        <width>180</width>
        <height>40</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="CheckBtn3">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>270</y>
        <width>180</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Sprawdź</string>
      </property>
     </widget>
     <widget class="QPushButton" name="jpnMainMenu">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>130</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Menu Główne</string>
      </property>
     </widget>
     <widget class="QLabel" name="jpnWord">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>100</y>
        <width>281</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Monaco</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="jpnRes">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>210</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="styleSheet">
      <string notr="true">QLabel{
	color: rgb(22, 13, 116);
	border:none;
	font-size: 20px;
}
QPlainTextEdit{
	background-color: rgb(255, 244, 244)
}
QComboBox{
	background-color: rgb(255, 244, 244);
	color: rgb(22, 13, 116);
}</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string> Tworzenie testów</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="text_createtest_transl">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>270</y>
        <width>150</width>
        <height>50</height>
       </rect>
      </property>
      <property name="plainText">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>130</y>
        <width>150</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox{
	background-color:rgba(255, 136, 0, 255);
	
}
QComboBox:hover{
	background-color:rgb(255, 150, 255);
	
}
QComboBox::item {
	background-color:rgba(255, 136, 0, 255);
  }
QComboBox::item:selected {
      border: 1px solid #6a6ea9;
  }

  QComboBox::item:selected:!active {
      background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                  stop: 0 #ABAFE5, stop: 1 #8588B2);
  }

  QComboBox::item:selected:active {
      background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                  stop: 0 #6a6ea9, stop: 1 #888dd9);
  }

  QComboBox::item:hover {
      background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                  stop: 0 #FAFBFE, stop: 1 #DCDEF1);
  }</string>
      </property>
      <item>
       <property name="text">
        <string>Angielski</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Niemiecki</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>130</y>
        <width>150</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>  Język</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>270</y>
        <width>150</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string> Tłumaczenie</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>200</y>
        <width>150</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string> Wybierz plik</string>
      </property>
     </widget>
     <widget class="QPushButton" name="select_img_bt">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>200</y>
        <width>150</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>wybierz plik</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backBtn">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>410</y>
        <width>171</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Powrót </string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_img_bt">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>320</y>
        <width>150</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>Dodaj</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_img_watch">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>80</y>
        <width>221</width>
        <height>211</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>90</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>  Tytuł</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="testName">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>90</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>742</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
